lib/themes/dosomething/paraneue_dosomething/includes/preprocess.inc

```diff
@@ -665,7 +668,7 @@ function paraneue_dosomething_preprocess_user_profile(&$vars) {
         'title' => $value['link'],
         'image' => $image,
         'description' => $value['call_to_action'],
-        'url' => dosomething_global_url($value['path_alias'], ['fragment' => 'prove']),
+        'url' => $GLOBALS['base_url'] . '/' . dosomething_global_get_prefix_for_language($language) . '/' . drupal_get_path_alias('node/' . $value['nid'], $language),
```

Comment:
Not sure if using `$GLOBALS['base_url']` is correct. It's better to use `url()` function.